pkgs/tools/audio/opl3bankeditor/opn2bankeditor.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, opl3bankeditor, fetchFromGitHub, fetchpatch }:
+
+opl3bankeditor.overrideAttrs (oldAttrs: rec {
+  version = "1.3-beta";
+  pname = "OPN2BankEditor";
+
+  src = fetchFromGitHub {
+    owner = "Wohlstand";
+    repo = pname;
+    rev = version;
+    sha256 = "0blcvqfj1yj6cmm079aw4jdzv3066jxqy9krp268i6cl2b3bmwvw";
+    fetchSubmodules = true;
+  };
+
+  postFixup = ''
+    install -Dm755 opn2_bank_editor $out/bin/opn2_bank_editor
```

Comment:
Does it install anything useful during installPhase? Maybe installPhase or postInstall is more appropriate?

Reply:
@etu Yes, it installs icons, desktop item and translations. Missing binary is a bug and that will be fixed with next release, at which I remove that stage. Moved to postInstall, thanks!